apps/archive/commands.py

```diff
@@ -28,6 +28,16 @@
 logger = logging.getLogger(__name__)
 
 
+def log_exeption(fn):
```

Comment:
I don't get why it's needed, there is no logging otherwise?

Reply:
should be logged here https://github.com/superdesk/superdesk-core/blob/master/superdesk/__init__.py#L57-L64

Reply:
we have two calls
```
+        self._remove_expired_publish_queue_items()
 +        self._remove_expired_items(now)
```
in previous version if the first fails second one never call, so I wrapped these function to be sure that second function will be called

Reply:
I see, would be good imo to put this into the comment

Reply:
comment is here
```diff
+        # next functions wrapped with log_exeption, so no try there
```